### PR TITLE
CI

### PR DESCRIPTION
**Pull request by mdehoog**
Example run: https://github.com/mdehoog/eip4844-interop/runs/6902550384

### CODE REVIEW

File: blob_tx_generator/blob.js [hunk 1]
```diff
@@ -109,6 +105,38 @@ async function run(data) {
     console.log(`sending to ${account.address}`)
     const res = await axios.post("http://localhost:8545", req)
     console.log(res.data)
+    if (res.data.error) {
+        process.exit(1)
+    }
+
+    let blob_kzg = null
+    try {
+        let start = (await axios.get("http://localhost:3500/eth/v1/beacon/headers")).data.data[0].header.message.slot - 1
```
Comment (mdehoog) [inline blob_tx_generator/blob.js:119]: noticed that `getBlockNumber` is out of sync with the beacon slots? not sure why

File: blob_tx_generator/blob.js [hunk 2]
```diff
@@ -2,9 +2,7 @@ const ethers = require("ethers")
 const axios = require('axios')
 
 const input = process.argv[2]
-const privateKey = "0x45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
 const provider = new ethers.providers.JsonRpcProvider("http://localhost:8545")
-const wallet = new ethers.Wallet(privateKey, provider)
 
 const BYTES_PER_FIELD_ELEMENT = 32
 const FIELD_ELEMENTS_PER_BLOB = 1016
```
Comment (Inphi) [inline blob_tx_generator/blob.js]: unrelated but this is a bug. Please update to `4096`

File: blob_tx_generator/blob.js [hunk 3]
```diff
@@ -78,15 +76,14 @@ async function estimateGas(tx) {
...
+        if (num >= 9) {
             break
         }
         console.log(`waiting for eip4844 proc.... bn=${num}`)
         await sleep(1000)
     }
     let blobs = get_blobs(data)
     console.log("number of blobs is " + blobs.length)
-    const bb = blobs.toString('binary')
     const blobshex = blobs.map((x) => { x.toString('hex') })
```
Comment (Inphi) [inline blob_tx_generator/blob.js]: another old bug here. Each hex encoded blob should include the `0x` prefix:
```js
const blobshex = blobs.map((x) => "0x" + x.toString('hex'))
```

File: blob_tx_generator/blob.js (hunk 1 above)
Comment (Inphi) [reply blob_tx_generator/blob.js:119]: Hmm. How far out of sync?

Comment (mdehoog) [reply blob_tx_generator/blob.js:119]: Estimates, but when testing locally, `getBlockNumber` was returning block 280 when the beacon chain was returning 390. This likely wouldn't happen in CI though as we don't wait that long

**Review by Inphi — APPROVED** [review]
Nice! Thank you!!

**Pull request merged**